### PR TITLE
feat: config storybook

### PR DESCRIPTION
**Pull request by chungnguyenhoang**
image
 
image

Screenshot 2023-02-07 at 16 55 26

when run script `yarn storybook`, we will test Component on Web.
we can test on simulator with `yarn android` or `yarn ios`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,19 +51,37 @@
...
+    "@storybook/addon-essentials": "^6.5.14",
+    "@storybook/addon-links": "^6.5.14",
+    "@storybook/addon-ondevice-actions": "next",
+    "@storybook/addon-ondevice-backgrounds": "next",
+    "@storybook/addon-ondevice-controls": "next",
+    "@storybook/addon-ondevice-notes": "next",
+    "@storybook/addon-react-native-web": "^0.0.19",
+    "@storybook/react-native": "next",
+    "@storybook/react": "^6.5.14",
+    "@storybook/testing-library": "^0.0.13",
```
Comment (loc-nguyenthien) [inline package.json:65]: Why use too many addons
Will using `next` cause any issue when those version goes higher than the rest?

File: package.json [hunk 2]
```diff
@@ -48,19 +51,37 @@
...
     "eslint": "^8.4.1",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^28.1.1",
+    "metro-react-native-babel-preset": "0.72.3",
     "pod-install": "^0.1.0",
     "prettier": "^2.0.5",
     "react": "18.1.0",
+    "react-dom": "^18.2.0",
+    "react-native-web": "^0.18.10",
```
Comment (loc-nguyenthien) [inline package.json:84]: Try using storybook for React Native

File: src/Button/Button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// stories/MyButton.stories.tsx
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {MyButton} from './Button'
```
Comment (loc-nguyenthien) [inline src/Button/Button.stories.tsx:5]: Should use a more descriptive name for `MyButton`

Comment (chungnguyenhoang) [reply src/Button/Button.stories.tsx:5]: We can read more Docs in https://storybook.js.org/tutorials/intro-to-storybook/react-native/en/simple-component/

File: package.json (hunk 2 above)
Comment (chungnguyenhoang) [reply package.json:104]: Storybook on Device Test Component  using `react-dom ` for Navigator 
Storybook on Browers Test Component  using `react-native-web ` for lib `@storybook/addon-react-native-web`

File: package.json (hunk 1 above)
Comment (chungnguyenhoang) [reply package.json]: the Storybook I'm using `V6.5`. 
but lib `@storybook/react-native` support for version `6.5` still be Beta-Version so i need using `next`

File: src/Button/Button.stories.tsx (hunk 3 above)
Comment (loc-nguyenthien) [reply src/Button/Button.stories.tsx:5]: Skip for now

**Review by loc-nguyenthien — APPROVED** [review]

**Review by baonguyenhsts — APPROVED** [review]

**Review by ngochuyduong — APPROVED** [review]

**Review by TranQuangPhi — APPROVED** [review]

**Review by loido — APPROVED** [review]

**Pull request closed without merging**